source/puppet/3.8/reference/lang_reserved.markdown

```diff
@@ -60,6 +60,14 @@ The following words are reserved for future use:
 * `type`
 * `private`
 
+The following words are reserved when using the future parser:
```

Comment:
There's actually a separate file you'll need to update as well: future_lang_reserved. There are two copies of the language docs in 3.8, since there are two copies of the language.